### ISSUE TITLE
Provide a web based formatting tool

### ISSUE THREAD

**Issue by pca006132** [hugheaves/scadformat#3]
It seems possible to compile go libraries to wasm, and run it on the web. That way you can have a github page that does the formatting. Not familiar with go though, not sure what is required here.

**Comment by hugheaves** [comment]
Good idea, and thanks for the suggestion! I'll investigate and see what it would take to implement.
